modules/nw-ptp-configuring-linuxptp-services-as-grandmaster-clock.adoc

```diff
@@ -73,10 +86,48 @@ $ oc logs linuxptp-daemon-74m2g -n openshift-ptp -c linuxptp-daemon-container
 .Example output
 [source,terminal]
 ----
-ts2phc[94980.334]: [ts2phc.0.config] nmea delay: 98690975 ns
-ts2phc[94980.334]: [ts2phc.0.config] ens3f0 extts index 0 at 1676577329.999999999 corr 0 src 1676577330.901342528 diff -1
-ts2phc[94980.334]: [ts2phc.0.config] ens3f0 master offset         -1 s2 freq      -1
-ts2phc[94980.441]: [ts2phc.0.config] nmea sentence: GNRMC,195453.00,A,4233.24427,N,07126.64420,W,0.008,,160223,,,A,V
-phc2sys[94980.450]: [ptp4l.0.config] CLOCK_REALTIME phc offset       943 s2 freq  -89604 delay    504
-phc2sys[94980.512]: [ptp4l.0.config] CLOCK_REALTIME phc offset      1000 s2 freq  -89264 delay    474
+ptp4l[69011.473]: [ptp4l.1.config] master offset          1 s2 freq   -7272 path delay       528
+I0816 13:51:13.521655 1781035 pmc.go:27] pmc read config from /var/run/ptp4l.1.config
+I0816 13:51:13.521742 1781035 pmc.go:28] pmc run command: GET PARENT_DATA_SET
+ptp4l[69011.474]: [ptp4l.0.config] master offset          1 s2 freq  -13442 path delay       528
+I0816 13:51:13.521974 1781035 pmc.go:27] pmc read config from /var/run/ptp4l.0.config
+I0816 13:51:13.521995 1781035 pmc.go:28] pmc run command: GET PARENT_DATA_SET
+I0816 13:51:13.524024 1781035 pmc.go:40] pmc result: sending: GET PARENT_DATA_SET
+    b49691.fffe.a3f3f0-0 seq 0 RESPONSE MANAGEMENT PARENT_DATA_SET
+        parentPortIdentity                    3c2c30.ffff.670e00-11
+        parentStats                           0
+        observedParentOffsetScaledLogVariance 0xffff
+        observedParentClockPhaseChangeRate    0x7fffffff
+        grandmasterPriority1                  128
+        gm.ClockClass                         6
+        gm.ClockAccuracy                      0x21
+        gm.OffsetScaledLogVariance            0x4e5d
+        grandmasterPriority2                  128
+        grandmasterIdentity                   001747.fffe.7015bc
 ----
++
+`gm.ClockClass  6` indicates that the T-GM is in locked mode and syncing correctly with the upstream GNSS source.
++
+.T-GM clock class values
+[cols="1,3", width="90%", options="header"]
+|====
+|Clock class value
+|Description
+
+|6
+|The grandmaster clock is connected to the Primary Reference Time Clock (PRTC) in locked mode.
+
+|140
+|The grandmaster clock is in holdover mode, is out of holdover specification, but is still traceable to the Category 1 frequency source.
+
+|150
```

Comment:
I think we are not using 160 and 150
Used clock class for T-GM in WPC 
```
| FREERUN                              | NA                | NA                | FREERUN  | 248
| HOLDOVER IN SPEC           | NA                | NA                | HOLDOVER | 7
| FREERUN OUT OF SPEC     | NA                | NA                | FREERUN  | 140
| LOCKED                                 | LOCKED            | LOCKED            | LOCKED   | 6
```

Reply:
okay , Looks like this section  explains all clock class regardless of what is implemented.I think we can leave it then

Reply:
I think better to have just what we have implemented.

Reply:
SO we implement 6, 7, 140, 248?

Reply:
yes for GM in 4.14